packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1847,6 +1850,7 @@ async function handler(
     status: "ACCEPTED",
     smsReminderNumber: booking?.smsReminderNumber || undefined,
     rescheduledBy: reqBody.rescheduledBy,
+    ...(assignmentReason ? { assignmentReason: [assignmentReason] } : {}),
```

Comment:
We're passing this as an array. In follow ups we will want to pass the whole history of the booking